homeassistant/components/august/manifest.json

```
@@ -5,5 +5,9 @@
   "requirements": ["py-august==0.25.2"],
   "dependencies": ["configurator"],
   "codeowners": ["@bdraco"],
+  "dhcp": [
+      {"hostname":"connect","macaddress":"D86162*"},
```

Since we use `fnmatch` for hostname, should this be
```suggestion
      {"hostname":"connect*","macaddress":"D86162*"},
```
?

---

Surprisingly, it actually always `connect` for all the august devices.